libs/zzz/formula/src/data/disc/sheets/ShadowHarmony.ts

```diff
@@ -1,49 +1,36 @@
-import { cmpGE } from '@genshin-optimizer/pando/engine'
+import { cmpGE, prod } from '@genshin-optimizer/pando/engine'
 import type { DiscSetKey } from '@genshin-optimizer/zzz/consts'
-import {
-  allBoolConditionals,
-  allListConditionals,
-  allNumConditionals,
-  enemyDebuff,
-  own,
-  ownBuff,
-  registerBuff,
-  teamBuff,
-} from '../../util'
-import { entriesForDisc, registerDisc } from '../util'
+import { allNumConditionals, own, ownBuff, registerBuff } from '../../util'
+import { registerDisc } from '../util'
 
 const key: DiscSetKey = 'ShadowHarmony'
 
 const discCount = own.common.count.sheet(key)
 const showCond4Set = cmpGE(discCount, 4, 'infer', '')
 
-// TODO: Add conditionals
-const { boolConditional } = allBoolConditionals(key)
-const { listConditional } = allListConditionals(key, ['val1', 'val2'])
-const { numConditional } = allNumConditionals(key, true, 0, 2)
+const { stacks } = allNumConditionals(key, true, 0, 3)
 
 const sheet = registerDisc(
   key,
   // Handle 2-set effects
-  entriesForDisc(key),
+  // entriesForDisc(key) // currrently mapped to dmg_, which is not specific enough.
+  [
+    ...ownBuff.initial.dmg_.addWithDmgType('dash', cmpGE(discCount, 2, 0.15)),
+    ...ownBuff.initial.dmg_.addWithDmgType(
```

Comment:
Should be in registerbuff with cond

Reply:
this is for the 2p effect, looks to be exactly what `entriesForDisc` does

Reply:
I'll change entries for disc then, it should also register buff